packages/composable/src/FormComposable/docs.tsx

```diff
@@ -8,5 +8,6 @@ export interface InputTypes {
 }
 
 export const FormField = (props: InputTypes & CommonFieldProps) => {
+  console.log(props)
```

Comment:
Do we need to log here?

Reply:
It's a fake component never used in the code only usefull in the doc to display the formField props table in storybook documentation:
![image](https://github.com/komune-io/fixers-g2/assets/47100280/9dd39110-5fbc-49c4-b172-ffee8832ee1a)

I removed it and let the props in the component to make use of it avoiding typescript from triggering a warning